tests/test_main.py

```diff
@@ -343,3 +344,17 @@ def test_format_linter_report(errors_linter, errors_mate):
         f"Function 'format_linter_report' should return {errors_mate} "
         f"when 'errors' equals to {errors_linter}"
     )
+
+
+def test_comment_deleted():
+    lines = inspect.getsource(main)
+    assert "# write your code here" not in lines, (
+        "Remove the unnecessary" " comment '# write your code here'"
+    )
+
+
+def test_double_quotes_instead_of_single():
+    lines = inspect.getsource(main)
+    assert "'" not in lines, (
+        'You have to use a double quotes "" instead' " of single ''"
```

Comment:
This test `test_double_quotes_instead_of_single` is not necessary. The choice between single and double quotes depends on the project's code style guide. It's not a critical issue that affects the functionality of the code.